### ISSUE TITLE
lidar intensity normalization

### ISSUE THREAD

**Issue by josh-wende** [NVIDIA-AI-IOT/Lidar_AI_Solution#230]
Hi, thanks for this project.

I have my own lidar scans I'm trying to use with the pretrained models, but the raw intensity values I have range from 1 to about 5600, with an average of about 500. I'm trying to figure out how to best normalize these to match the distribution of the training data. Is there a "correct" answer to this? Looking at the example data, it seems like the intensities should be scaled to [0, 255]?

I appreciate any insight.

**Comment by hopef** [comment]
`(intensity - mean) / std` would be a good consideration for you.
